### PR TITLE
bears: Add metadata

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Adds values to the sets and bears contain
metadata about them.

### CODE REVIEW

File: bears/c_languages/CPPCheckBear.py [hunk 1]
```diff
@@ -19,6 +19,13 @@ class CPPCheckBear:
     """
 
     LANGUAGES = {"C", "C++"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'https://groups.google.com/forum/#!forum/coala-devel'}
```
Comment (sils1297) [inline bears/c_languages/CPPCheckBear.py]: that's no e-mail address. Rather use `coala-devel@googlegroups.com`

File: bears/c_languages/CPPCheckBear.py [hunk 2]
```diff
@@ -19,6 +19,13 @@ class CPPCheckBear:
...
+    MAINTAINERS = {'Lasse Schuirmann', 'Fabian Neuschmidt', 'Mischa Kr\xfcger'}
```
Comment (sils1297) [inline bears/c_languages/CPPCheckBear.py]: isn't this the coala devs as well? Could we maybe default it to  the AUTHORS if not given?

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala-bears/pull/524/commits/db5ef317e38cd455a19ceecaa0a8b17a4ee60ae4 needs work

File: bears/c_languages/CPPCheckBear.py (hunk 2 above)
Comment (Adrianzatreanu) [inline bears/c_languages/CPPCheckBear.py]: this is what the current setup.py says .. you want me to make it equal to authors if not given? in Bear.py?

**Comment by sils1297** [comment]
ack 9e7547d

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**